### DISCUSSION TITLE
第2节课笔记(12班)

### DISCUSSION THREAD

**Discussion by fanqiNO1** [InternLM/Tutorial#89] [笔记]
12班第2节课的笔记交在这里哦

**Comment by jujimeizuo** [comment]
https://note.jujimeizuo.cn/llm/internlm/lec2/

**Comment by HuangXinzhe** [comment]
https://blog.csdn.net/qq_29787929/article/details/135431064

**Comment by JiangXiaobai00** [comment]
https://d0elb2210mg.feishu.cn/docx/G1nndwPdAomTYmxHsLhcnoRinSb?from=from_copylink

**Comment by ltstar** [comment]
[课程2.md](https://github.com/InternLM/tutorial/files/13853417/2.md)

**Comment by Silflame** [comment]
https://www.yuque.com/wayne-o8axv/hdcch1/kxgcyg0pzcpe92pe

**Comment by ChengaoJ** [comment]
https://blog.csdn.net/weixin_43193528/article/details/135447175

**Comment by cheng221** [comment]
https://spice-spice-829.notion.site/1a1dc7130573423eaccd3c58c566e2b0?pvs=4

**Comment by hxy-tsy** [comment]
[第二节课.md](https://github.com/InternLM/tutorial/files/13871180/default.md)

**Comment by siuserjy** [comment]
https://blog.csdn.net/weixin_46334272/article/details/135459509

**Comment by jiangzf93** [comment]
https://blog.csdn.net/weixin_42890306/article/details/135490879

**Comment by deseijily** [comment]
https://j1b8l2ps5tm.feishu.cn/docx/IC68d7h1mon1DaxFhqics8Thn8e

**Comment by ShenFeilang** [comment]
https://ink-neon-bce.notion.site/InternLM-a9204f43e7884065b690f7109cafee35?pvs=4
